db/migrate/20150825092700_create_speech.rb

```diff
@@ -0,0 +1,16 @@
+class CreateSpeech < ActiveRecord::Migration
+  def change
+    create_table :speeches do |t|
+      t.string :name
+      t.text :description
+      t.belongs_to :user, index: true
+      t.string :slides
```

Comment:
I feel "slides" is too concrete, how about general name "resources".

E.x. The speaker may upload source codes as his speech reference.